index.js

```diff
@@ -99,7 +81,7 @@ const displayPost = (post) => {
     </td>
     <td class='post-statistic'></td>
     <td class='post-statistic'></td>
-    <td class='post-statistic'>${displayRecentActivity()}</td>
+    <td class='post-statistic'>${displayRecentActivity(post.bumped_at)}</td>
```

Comment:
```suggestion
    <td class='post-statistic'>${formatDateDiff(Date.now(), post.bumped_at)}</td>
```
new Date(new Date()) == new Date() == new Date(Date.now())